compiler/src/iree/compiler/Dialect/HAL/Target/LLVMCPU/LLVMTargetOptions.cpp

```diff
@@ -49,6 +52,79 @@ struct HostDefaults {
   }
 };
 
+void ResolveCPUAndCPUFeatures(llvm::StringRef inCpu,
```

Comment:
style: LLVM uses camelCase for functions
this can also be static

Reply:
(this is in anonymous namespace)